### ISSUE TITLE
wishlist: run multiple files with the selected profile

### ISSUE THREAD

**Issue by rooiejoris** [daid/LegacyCura#136]
wishlist:
run multiple files with the selected profile
now i use a script, maybe nice to have a sort of interface where you can select the files and select the profile [where current profile is also an option]

cheers\joris

**Comment by daid** [comment]
Sort of like a "batch run" in Photoshop?

**Comment by rooiejoris** [comment]
i think so...
a file [or folder] selectbox and a choose profile box and then run...

**Comment by daid** [comment]
With an option to do X slices in parallel ;-)

**Comment by daid** [comment]
This feature was implemented in 12.08
